back-end/www/tests/location_tests.py

```diff
@@ -0,0 +1,183 @@
+from basic_tests import BasicTest
+from models.model_operations import location_operations
+from models.model_operations import answer_operations
+from models.model_operations import user_operations
+from models.model import db
+import unittest
+
+
+class LocationTest(BasicTest):
+    """Test case for locations."""
+
+    def setUp(self):
+        db.create_all()
+
+
```

Comment:
I know this is confusing. But for styling, functions should be separated by "one" blank line when the functions are in  a class. In other cases, functions should be separated by "two" blank lines.